csv-validator-ui/src/main/scala/uk/gov/nationalarchives/csv/validator/ui/CsvValidatorUi.scala

```diff
@@ -377,53 +370,63 @@ object CsvValidatorUi extends SimpleSwingApplication {
-      onClick(displayWait(
-        suspendUi = {
-          btnValidate.enabled = false
-          btnValidateMetadataOnly.enabled = false
-          this.peer.setCursor(Cursor.getPredefinedCursor(Cursor.WAIT_CURSOR))
-          this.progressBar.value = 0
-          this.progressBar.visible = true
-
-        },
-        action = {
-          txtArReport.text = ""
-          CsvValidatorUi.this.validate(
-            txtCsvFile.getText,
-            settingsPanel.csvEncoding,
-            txtCsvSchemaFile.getText,
-            settingsPanel.csvSchemaEncoding,
-            settingsPanel.maxCharsPerCell,
-            settingsPanel.failOnFirstError,
-            settingsPanel.pathSubstitutions,
-            settingsPanel.enforceCaseSensitivePathChecks,
-            Some(progress),
-            settingsPanel.validateUtf8,
-            settingsPanel.numOfLinesToDisplay,
-            skipFileChecks,
-            suffix
-          )
-        },
-        output = outputToReport,
-        resumeUi = {
-          btnValidate.enabled = true
-          btnValidateMetadataOnly.enabled = true
-          btnValidate.peer.setCursor(Cursor.getDefaultCursor)
-          btnValidateMetadataOnly.peer.setCursor(Cursor.getDefaultCursor)
-          //this.progressBar.visible = false
-        }
-      ))
+      val csvFilePath = txtCsvFile.getText
+      val csvSchemaFilePath = txtCsvSchemaFile.getText
+      val fileBoxesAreFilled = csvFilePath.nonEmpty && csvSchemaFilePath.nonEmpty
+
+      lazy val maxCharsPerCell = convertTextboxValueToInt(settingsPanel.maxCharsPerCell, "characters per column", outputToReport(clearOutput = true))
+      lazy val maxNumOfLines = convertTextboxValueToInt(settingsPanel.numOfLinesToDisplay, "errors to display", outputToReport(clearOutput = true))
+
+      if(fileBoxesAreFilled && maxCharsPerCell > 0 && maxNumOfLines > 0){
+        val suffix = if(skipFileChecks) " (Metadata Only)" else ""
+        displayWait(
+          suspendUi = {
+            btnValidate.enabled = false
+            btnValidateMetadataOnly.enabled = false
+            this.peer.setCursor(Cursor.getPredefinedCursor(Cursor.WAIT_CURSOR))
+            this.progressBar.value = 0
+            this.progressBar.visible = true
+
+          },
+          action = {
+            txtArReport.text = ""
+            CsvValidatorUi.this.validate(
+              csvFilePath,
+              settingsPanel.csvEncoding,
+              csvSchemaFilePath,
+              settingsPanel.csvSchemaEncoding,
+              maxCharsPerCell,
+              settingsPanel.failOnFirstError,
+              settingsPanel.pathSubstitutions,
+              settingsPanel.enforceCaseSensitivePathChecks,
+              Some(progress),
+              settingsPanel.validateUtf8,
+              maxNumOfLines,
+              skipFileChecks,
+              suffix
+            )
+          },
+          output = outputToReport(clearOutput = false),
```

Comment:
`false` is the default value for `clearOutput` so you can remove this from here or remove the default value.